packages/clerk-js/bundlewatch.config.json

```diff
@@ -1,8 +1,8 @@
 {
   "files": [
-    { "path": "./dist/clerk.js", "maxSize": "610kB" },
-    { "path": "./dist/clerk.browser.js", "maxSize": "70.16KB" },
-    { "path": "./dist/clerk.legacy.browser.js", "maxSize": "113KB" },
+    { "path": "./dist/clerk.js", "maxSize": "612kB" },
```

Comment:
![CleanShot 2025-06-18 at 13 21 54](https://github.com/user-attachments/assets/26766adf-a5f9-4ca3-b8ab-84e4f058bfc7)

The bundle increased around 4KB since I extracted the force-an-org flow outside of `OrganizationList`. It increased the general bundle, but we are lazy loading `SessionTasks` within `SignIn`. 

I think the tradeoff is worth considering. I didn't want to introduce another `if (sessionTasksContext)` within `OrganizationList` to hijack the logic in there, but rather keep it separate. 

Ideally, would be nice to use `Suspense` to only download the necessary after-auth steps according to `Enviroment` settings.